src/legacy/ui/public/management/_index.scss

```diff
@@ -1 +1 @@
-@import './components/index';
\ No newline at end of file
+@import '../../../../plugins/management/public/components/index';
```

Comment:
I think you could move this scss file to NP and import it from `ui/public/_index.scss`